client/me/help/help-courses/index.jsx

```diff
@@ -58,14 +60,21 @@ function getCourses() {
 	];
 }
 
+function mapDispatchToProps( dispatch ) {
+	// This function only adds a way of dispatching courses because we don't have another mechanism yet.
+	// Once the courses make it into the API this function should go away in preference for
+	// something linke <QueryHelpCourses />
+	return {
+		fetchCourses: () => dispatch( receiveHelpCourses( getCourses() ) )
```

Comment:
I know this is just temporary, but rather than receiving the courses here how about moving the temp data into a module and loading it as the default state in the reducer?

eg
```js
import defaultCourses from './courses';
// ...
export const items = createReducer( defaultCourses, {
	[ HELP_COURSES_RECEIVE ]: ( state, { courses } ) => courses
} );
```